### PR TITLE
helm changes for VCF cluster and VIP per Namespace

### PR DESCRIPTION
**Pull request by monotosh-avi**
- Also introduced a new flag vipPerNamespace so that this feature can be used outside a VCF cluster

### CODE REVIEW

File: helm/ako/templates/configmap.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ data:
   enableEVH: {{ .Values.AKOSettings.enableEVH | quote }}
   layer7Only: {{ .Values.AKOSettings.layer7Only | quote }}
   istioEnabled: {{ .Values.AKOSettings.istioEnabled | quote }}
+  vipPerNamespace: {{ .Values.AKOSettings.vipPerNamespace | quote }}
+  vcfCluster: {{ .Values.AKOSettings.vcfCluster | quote }}
```
Comment (sudswasavi) [inline helm/ako/templates/configmap.yaml:20]: Is there a reason we want to expose this flag in the AKO Configmap?

File: helm/ako/templates/statefulset.yaml [hunk 2]
```diff
@@ -68,6 +68,16 @@ spec:
               configMapKeyRef:
                 name: avi-k8s-config
                 key: istioEnabled
+          - name: VIP_PER_NAMESPACE
+            valueFrom:
+              configMapKeyRef:
+                name: avi-k8s-config
+                key: vipPerNamespace
+          - name: VCF_CLUSTER
```
Comment (sudswasavi) [inline helm/ako/templates/statefulset.yaml:76]: This may not be required in upstream AKO.

File: helm/ako/templates/configmap.yaml (hunk 1 above)
Comment (monotosh-avi) [reply helm/ako/templates/configmap.yaml:20]: without a flag it would be difficult to test internally

**Review by sudswasavi — APPROVED** [review]

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Review by sudswasavi — APPROVED** [review]

File: helm/ako/templates/configmap.yaml [hunk 3]
```diff
@@ -16,6 +16,7 @@ data:
   enableEVH: {{ .Values.AKOSettings.enableEVH | quote }}
   layer7Only: {{ .Values.AKOSettings.layer7Only | quote }}
   istioEnabled: {{ .Values.AKOSettings.istioEnabled | quote }}
+  vipPerNamespace: {{ .Values.AKOSettings.vipPerNamespace | quote }}
```
Comment (akshayhavile) [inline helm/ako/templates/configmap.yaml:19]: Do we need this change in AKO-operator context?

Comment (monotosh-avi) [reply helm/ako/templates/configmap.yaml:19]: Thanks, had missed that, added now.

**Review by akshayhavile — APPROVED** [review]

**Review by sudswasavi — APPROVED** [review]

**Pull request merged**